build.gradle

```diff
@@ -3,7 +3,7 @@ plugins {
   id 'checkstyle'
   id 'io.spring.dependency-management' version '1.0.5.RELEASE'
   id 'org.flywaydb.flyway' version '5.0.7'
-  id 'org.springframework.boot' version '1.5.10.RELEASE'
+  id 'org.springframework.boot' version '1.5.13.RELEASE'
```

Comment:
ah. i better investigate this update. `1.5.12` was introduce heck lot's of tomcat issues